backend/rates/rates.go

```diff
@@ -153,15 +128,14 @@ func (updater *RateUpdater) Last() map[string]map[string]float64 {
 }
 
 // LastForPair returns the conversion rate for the given (coin, fiat) pair. Returns an error if the
-// rates have not been fetched yet. `coinCode` values are the same as `coin.Code`.
-func (updater *RateUpdater) LastForPair(coinCode, fiat string) (float64, error) {
+// rates have not been fetched yet. `coinUnit` values are the same as `coin.Unit`.
+func (updater *RateUpdater) LastForPair(coinUnit, fiat string) (float64, error) {
```

Comment:
So, we now have the following list of exported methods:

- `Last`
- `LastForPair`
- `PriceAt`

The first one returns current instant rates, the other too "historical" but if I were to see them used anywhere in the codebase, I'd be totally confused which one is which and what the difference is.

I think it's time to give them better names to distinguish between /simple/price and historical rates.

Reply:
I am not sure what the confusion is. `Last` is latest price, `PriceAt` is price at a specific time. What would be better names`

`LatestPrice` and `HistoricalPrice`?

Reply:
updated names